### PR TITLE
fisher_swbd clean-up

### PR DESCRIPTION
**Pull request by keli78**
added several necessary files and modified run.sh of fisher_swbd recipe

### CODE REVIEW

**Comment by vijayaditya** [comment]
Were you able to reproduce the results with these changes ?

**Comment by keli78** [comment]
The code is still running. I'm not sure whether it can exactly reproduce the results.

**Comment by danpovey** [comment]
- *A* egs/swbd/s5/local/swbd1_data_download.sh
   <https://github.com/kaldi-asr/kaldi/pull/475/files#diff-7> (60)
   - *A* egs/swbd/s5b/local/swbd1_data_download.sh
   <https://github.com/kaldi-asr/kaldi/pull/475/files#diff-8> (60)

why have you added the above files?  If there is no script calling them,
what is the point?

**Comment by keli78** [comment]
Sorry for this misunderstanding. I have thought this is want you want me to
do. In your previous email, you said:"And please make the same change in
all the Switchboard recipes, in swbd/s5*/local/."
Ke

On Mon, Feb 1, 2016 at 3:28 PM, Daniel Povey <notifications@github.com>
wrote:

> - *A* egs/swbd/s5/local/swbd1_data_download.sh
> <https://github.com/kaldi-asr/kaldi/pull/475/files#diff-7> (60)
> - *A* egs/swbd/s5b/local/swbd1_data_download.sh
> <https://github.com/kaldi-asr/kaldi/pull/475/files#diff-8> (60)
>
> why have you added the above files? If there is no script calling them,
> what is the point?
>
>
> On Mon, Feb 1, 2016 at 1:17 PM, Ke Li <notifications@github.com> wrote:
>
> > The code is still running. I'm not sure whether it can exactly reproduce
> > the results.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/kaldi-asr/kaldi/pull/475#issuecomment-178107812>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/475#issuecomment-178171829>.
>



-- 
Ke Li
Dept. of Electrical and Computer Engineering
Johns Hopkins University
Email: kli26@jhu.edu

**Comment by danpovey** [comment]
OK- it looks like what I meant was to change from the old piconepress URL
to http://www.openslr.org/resources/5/switchboard_word_alignments.tar.gz.
You can fail over to the piconepress URL.  Any solution is OK that works,
including adding that new script- but there is no point adding that script
if your top-level script doesn't call it somewhere.  You need to find where
the script is downloading from picone press, and change something there.

Dan

File: egs/fisher_swbd/s5/cmd.sh [hunk 1]
```diff
@@ -5,28 +5,20 @@
 # (make sure your --num-jobs options are no more than
```
Comment (danpovey) [inline egs/fisher_swbd/s5/cmd.sh]: Please modify cmd.sh to read as follows (remove the plus signs at beginning of line though)
+# you can change cmd.sh depending on what type of queue you are using.
 +# If you have no queueing system and want to run on a local machine, you
 +# can change all instances 'queue.pl' to run.pl (but be careful and run 
 +# commands one by one: most recipes will exhaust the memory on your
 +# machine).  queue.pl works with GridEngine (qsub).  slurm.pl works
 +# with slurm.  Different queues are configured differently, with different
 +# queue names and different ways of specifying things like memory;
 +# to account for these differences you can create and edit the file
 +# conf/queue.conf to match your queue's configuration.  Search for
 +# conf/queue.conf in http://kaldi-asr.org/doc/queue.html for more information,
 +# or search for the string 'default_config' in utils/queue.pl or utils/slurm.pl.
 +
 +export train_cmd="queue.pl --mem 4G"
 +export decode_cmd="queue.pl --mem 4G"
 +export mkgraph_cmd="queue.pl --mem 8G"

**Comment by danpovey** [comment]
@naxingyu, I have accepted this commit and merged it into the chain branch, so the fisher_swbd recipe should work now.  If you have free cycles, it would be nice if you could help us tune the chain models for the fisher_swbd recipe.

**Comment by naxingyu** [comment]
OK.

在 2016/2/7 6:29, Daniel Povey 写道:

**Comment by naxingyu** [comment]
I'm on holiday now. I can work on it a week later if it is ok.
X.

在 2016/2/7 6:29, Daniel Povey 写道:

**Comment by danpovey** [comment]
sure.

**Pull request merged**